sentry-transport.js

```diff
@@ -6,6 +6,7 @@ var util = require('util'),
 var Sentry = winston.transports.Sentry = function (options) {
 
   this._dsn = options.dsn || '';
+  this._globalTags = options.tags || {};
```

Comment:
Do you mind if I also change the name of the field on the options object to globalTags? That's even more important as this is the API of the module.